lib/exd/api/crud.ex

```diff
@@ -78,43 +78,16 @@ defmodule Exd.Api.Crud do
   end
 
   defp get_one(api, %{"id" => id} = params) do
-    repo(api).get(model(api), id) |> save |> load(api, params)
+    data = model(api) |> Ecdo.query(Map.put(params, "where", [{:==, "id", id}])) |> repo(api).one |> save
+    unless_error(data, api, data)
   end
   defp get_one(api, %{"name" => name} = params) do
-    repo(api).get_by(model(api), name: name) |> save |> load(api, params)
+    data = model(api) |> Ecdo.query(Map.put(params, "where", [{:==, "name", name}])) |> repo(api).one |> save
```

Comment:
Why not stay with `get_by(model(api), name: name)` here? The same for `repo(api).get(model(api), id)` as we exactly know the query, which we are doing. For elixir language query, we should use ecto directly.